### ISSUE TITLE
Code Signing Error when loading the binary version of BPCells v0.2 (BPCells_0.2.0.tgz(r-4.4-arm64)) 

### ISSUE THREAD

**Issue by akhst7** [bnprks/BPCells#155]
Hi Ben, 

I run into the Apple's code signing error when loading BPCells_0.2.0.(r-4.4-arm64)

```
library(BPCells)
Error: package or namespace load failed for ‘BPCells’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '/Volumes/Document/4.3.X_Rlib/BPCells/libs/BPCells.so':
  dlopen(/Volumes/Document/4.3.X_Rlib/BPCells/libs/BPCells.so, 0x0006): tried: '/Volumes/Document/4.3.X_Rlib/BPCells/libs/BPCells.so' (code signature in <294A657F-219D-3EB2-AFE6-6CC754DEF5D2> '/Volumes/Document/4.3.X_Rlib/BPCells/libs/BPCells.so' not valid for use in process: library load disallowed by system policy), '/System/Volumes/Preboot/Cryptexes/OS/Volumes/Document/4.3.X_Rlib/BPCells/libs/BPCells.so' (no such file), '/Volumes/Document/4.3.X_Rlib/BPCells/libs/BPCells.so' (code signature in <294A657F-219D-3EB2-AFE6-6CC754DEF5D2> '/Volumes/Document/4.3.X_Rlib/BPCells/libs/BPCells.so' not valid for use in process: library load disallowed by system policy)
```
![Screenshot 2024-11-01 at 11 19 06 AM](https://github.com/user-attachments/assets/a5ca946d-25b1-470b-9949-62433c1dae92)

I appreciate if you can fix this.  

Thanks.

**Comment by bnprks** [comment]
Hi Aki, is this coming up after installing BPCells from r-universe? We're not really able to modify the build pipeline for that as it is controlled by the r-universe devs, so there's not much we can do easily to add code signing for Mac users.

I don't currently use a mac, but my recollection is that right after you get an error like this, if you go into your system security settings there's usually a button you can click to allow that specific unsigned binary to run and the next time you try the action it should work okay.

If you're not comfortable about this from a security perspective, building from source via github is probably the best bet.

-Ben

**Comment by akhst7** [comment]
Hi Ben, 

I figured this one out but it is a temporary solution since current OSX runs ACL and I could only temporarily disable this.  Anyway, it works now.   I will have to contact r-universe to properly report a code sign.  

I checked to make sure  nomalicious codes are present within their binary, which happens a lot lately.  

For compiling the latest BPCells in my armMac, I really need to cook it a bit further.   

Thanks anyway.  

Aki
